cernopendata/modules/theme/assets/semantic-ui/scss/search.scss

```diff
@@ -9,3 +9,20 @@
     text-decoration: none;
   }
 }
+
+.result-options {
+  margin-top: 20px;
+}
+
+.four.wide.column {
```

Comment:
Works nicely. I guess this will become a problem when we try to make the web responsive. I think we should open an issue on `inveniosoftware/React-SearchKit` to solve it upstream, what do you think?